mohamad-dakdouk/week5/mon/D21-games-hw/app/views/magic/question.html.erb

```diff
@@ -0,0 +1,3 @@
+<h3>Welcome to Magic 8</h3>
+
+<p>Submit your question into the tab bar to get an answer</p>
```

Comment:
don't get users to write in the web address bar. this is bad practice. 

You should have a `form` or `input` (with `name=query`) and then you can do something like `@question = params[:query]` and display the question

Reply:
You have already done this for secret number. rinse and repeat for magic 8